docs/conf.py

```diff
@@ -14,18 +14,46 @@
 
 from datetime import datetime
 import os
+from sphinx.directives.code import CodeBlock
 import sphinx_rtd_theme
 import sys
 
 
+# https://stackoverflow.com/questions/44761197/how-to-use-substitution-definitions-with-code-blocks
+class SubstitutionCodeBlock(CodeBlock):  # type: ignore
+  """
+  Similar to CodeBlock but replaces placeholders with variables.
+  """
+
+  def run(self):
+    """
+    Replace placeholders with given variables.
+    """
+    app = self.state.document.settings.env.app
+    new_content = []
+    self.content = self.content  # type: List[str]
```

Comment:
I'm a bit confused by this line :) Is this self-assigning to establish some additional type safety?

Reply:
Like a good developer, I just copied this code from stack overflow without looking at it. I will remove this line. :)